File: back.py
from fastapi import FastAPI
from fastapi.responses import PlainTextResponse
import uvicorn

app = FastAPI()

@app.get("/")
async def root():
    return PlainTextResponse("Hello from the backend (FastAPI app)")

if __name__ == "__main__":
    uvicorn.run(app, host="127.0.0.1", port=8000)

File: back.js
const express = require('express');
const app = express();
const PORT = 3000;

app.get('/', (req, res) => {
  res.send('Hello from the backend (Node.js app)');
});

app.listen(PORT, '127.0.0.1', () => {
  console.log(`Backend server running at http://localhost:${PORT}`);
});
